charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/pipeline-install-config-kubeflow-ConfigMap.yaml

```diff
@@ -4,19 +4,19 @@ data:
   DEFAULT_CACHE_STALENESS: ''
   MAXIMUM_CACHE_STALENESS: ''
   appName: pipeline
-  appVersion: 2.0.0-alpha.6
+  appVersion: 2.0.0-alpha.7
   autoUpdatePipelineDefaultVersion: 'true'
   bucketName: '{{ .Values.s3.bucketName }}'
   cacheDb: cachedb
   cacheImage: gcr.io/google-containers/busybox
   cacheNodeRestrictions: 'false'
   cronScheduleTimezone: UTC
-  dbHost: testa.c6uzclpednto.ca-central-1.rds.amazonaws.com
```

Comment:
why didnt the unit test catch this?

Reply:
this was from some of Ryan's previous IRSA chart, he accidentally left hard-coded values

Reply:
sure, shouldnt the helmify unit test fail after generation?